### ISSUE TITLE
  How to conveniently set other languages？

### ISSUE THREAD

**Issue by CrystalWindSnake** [zauberzeug/nicegui#1215]
### Description

```python

ui.add_body_html(
    """
<script>
  Quasar.lang.set(Quasar.lang.zhCN)
</script>
"""
)

@ui.page("/test")
def page_test():
    ui.date("2023-01-01")

ui.date("2023-01-01")
ui.run(language="zh-CN")
```

This code does not work.

**Comment by falkoschindler** [comment]
Oh, thanks for spotting this, @CrystalWindSnake!
It's actually a bug with language codes containing a region ("CN").
I already fixed it in commit [6f81f32](https://github.com/zauberzeug/nicegui/commit/6f81f3241777a9d87edc24ea12fcb2eb8eeca2a5). It will be part of the next release.

**Comment by CrystalWindSnake** [comment]
@falkoschindler Thank you so much for your help
